youtube_dl/extractor/twentymin.py

```diff
@@ -77,8 +92,14 @@ def _real_extract(self, url):
                 r'^20 [Mm]inuten.*? -', '', self._og_search_title(webpage)), ' - News')
 
         if not video_id:
+            params = self._html_search_regex(
+                r'<iframe[^>]+src="(?:https?:)?//www\.20min\.ch/videoplayer/videoplayer\.html\?params=(.+?[^"])"',
+                webpage, '20min embed URL', default=None)
             video_id = self._search_regex(
-                r'"file\d?"\s*,\s*\"(\d+)', webpage, 'video id')
+                r'.*videoId@(\d+)',
+                params, 'Video Id', default=None) if params is not None else ''
+            if not video_id:  # the article does not contain a video
```

Comment:
You can't know whether it's an article with no video or video id pattern just changed. Original approach should be reverted.